FogCreek/Glitch-Community
# luminous-buggy: add renderAsPlaintext option to Markdown option

* [luminous-buggy.glitch.me](https://luminous-buggy.glitch.me)
* [manuscript](https://glitch.manuscript.com/f/cases/3328313/Raw-markdown-rendered-in-add-project-to-collection-pop-and-maybe-elsewhere)

## Changes:
This adds a `renderAsPlaintext` prop to the `Markdown` component.

This was necessary because sometimes markdown content is rendered inside a button or otherwise clickable element, and we don't want the user to click a rendered markdown link

**Before**
<img width="270" alt="markdown1" src="https://user-images.githubusercontent.com/7584833/56523005-96787380-650c-11e9-9b57-1ce6a1549aa5.png">

**After**
<img width="270" alt="markdown2" src="https://user-images.githubusercontent.com/7584833/56523015-99736400-650c-11e9-973d-91f9faadfe1c.png">


## How To Test:
Open the add project to collection pop and confirm that collection descriptions are rendered as plaintext with no raw markdown.

## Feedback I'm looking for:
- Is this approach sound? Are there any weird security side effects I'm not thinking of?
- Should we move the `stripHtml` helper to `/utils`?
- Does this break the `Markdown` component in unexpected ways?
